src/player.py

```diff
@@ -1,2 +1,29 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+
+    def __init__(self, name, current_room):
+
+        self.name = name
+        self.current_room = current_room
+        self.items = []
+
+    def search_inventory(self, item):
```

Comment:
Solid work creating a `search_inventory` method. It's clear, concise, and readable.